### PR TITLE
Upgrade to v2.0.0 of Pulumi

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -47,7 +47,7 @@ build:: provider
 
 provider:: generate_schema
 	cd provider && go generate cmd/${PROVIDER}/main.go
-	cd provider && go install -ldflags "-X github.com/pulumi/pulumi-${PACK}/provider/v2/pkg/version.Version=${VERSION}" ${PROJECT}/provider/v2/cmd/${PROVIDER}
+	cd provider && go install -ldflags "-X github.com/pulumi/pulumi-${PACK}/provide/v2/pkg/version.Version=${VERSION}" ${PROJECT}/provider/v2/cmd/${PROVIDER}
```
Comment (mikhailshilkov) [inline Makefile:50]: that doesn't look right?

File: examples/virtualappliance/package.json [hunk 2]
```diff
@@ -1,16 +1,14 @@
...
+        "@pulumi/aws": "^2.0.0",
+        "@pulumi/pulumi": "^2.0.0",
+        "@pulumi/random": "^2.0.0",
         "@types/node-fetch": "^2.1.4",
-        "node-fetch": "^2.3.0"
-    },
-    "peerDependencies": {
-        "@pulumi/f5bigip": "dev"
+        "node-fetch": "^2.3.0",
+        "@pulumi/f5bigip": "^2.0.0"
```
Comment (mikhailshilkov) [inline examples/virtualappliance/package.json:12]: Why remove `peerDependencies`? I think we use them elsewhere (e.g. https://github.com/pulumi/pulumi-aws/blob/master/examples/minimal/package.json#L18)

File: Makefile (hunk 1 above)
Comment (stack72) [reply Makefile]: fixed!

File: examples/virtualappliance/package.json (hunk 2 above)
Comment (stack72) [reply examples/virtualappliance/package.json]: same comment as before - we are moving away from these - I will be sending a PR to the AWS repo to kill off peerDependencies so we can get some upgrade tests going

**Pull request merged**